### PR TITLE
PMM-8271 PITR backups for Mongo

### PR DESCRIPTION
**Pull request by fabio-silva**
**What this PR does / why we need it**: Adds PITR backups for MongoDB

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-8271

### CODE REVIEW

File: public/app/percona/backup/components/AddBackupModal/AddBackupModal.utils.ts [hunk 1]
```diff
@@ -59,11 +59,12 @@ export const toFormBackup = (backup: Backup | ScheduledBackup | null): AddBackup
       startMinute: [{ value: 0, label: '00' }],
       logs: false,
       active: true,
+      vendor: (null as unknown) as Databases,
```
Comment (tiagomotasantos) [inline public/app/percona/backup/components/AddBackupModal/AddBackupModal.utils.ts:62]: Can the type for `vendor` be `string | null`? So there won't be the need to cast to `unknown`

**Review by tiagomotasantos — CHANGES_REQUESTED** [review]

File: public/app/percona/backup/components/AddBackupModal/AddBackupModal.utils.ts (hunk 1 above)
Comment (fabio-silva) [reply public/app/percona/backup/components/AddBackupModal/AddBackupModal.utils.ts]: Good one 👍

**Review by tiagomotasantos — DISMISSED** [review]

File: public/app/percona/backup/components/AddBackupModal/AddBackupModal.types.ts [hunk 2]
```diff
@@ -20,11 +20,11 @@ export interface SelectableService {
 
 export interface AddBackupFormProps {
   id: string;
-  service: SelectableValue<SelectableService>;
+  service: null | SelectableValue<SelectableService>;
```
Comment (nicolalamacchia) [inline public/app/percona/backup/components/AddBackupModal/AddBackupModal.types.ts:23]: nit: I think it looks better to have `null` after the `|`, it reads better to me

**Review by nicolalamacchia — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

File: public/app/percona/backup/components/AddBackupModal/AddBackupModal.types.ts (hunk 2 above)
Comment (fabio-silva) [reply public/app/percona/backup/components/AddBackupModal/AddBackupModal.types.ts]: 👍

**Review by nicolalamacchia — APPROVED** [review]

**Pull request merged**